modules/ppcp-api-client/src/Endpoint/BillingAgreementsEndpoint.php

```diff
@@ -121,7 +121,7 @@ public function create_token( string $description, string $return_url, string $c
 	 */
 	public function reference_transaction_enabled(): bool {
 		try {
-			if ( get_transient( 'ppcp_reference_transaction_enabled' ) === true ) {
+			if ( get_transient( 'ppcp_reference_transaction_enabled' ) === '1' ) {
```

Comment:
hm, I think we save `true`, so it's not very obvious for it to become `'1'`.
Maybe it would be better to use `wc_string_to_bool`.